skillserver/echo.go

```diff
@@ -52,11 +59,13 @@ func (this *EchoRequest) GetSlotValue(slotName string) (string, error) {
 	return "", errors.New("Slot name not found.")
 }
 
+// Deprecated: Please use the github.com/mikeflynn/go-alexa/customskill package
 func (this *EchoRequest) AllSlots() map[string]EchoSlot {
```

Comment:
I didn't add this to `customskill`. I'm not convinced we need these little helper functions.